src/femr/labelers/omop_inpatient_admissions.py

```diff
@@ -143,11 +148,28 @@ def __init__(
         self.prediction_time_adjustment_func = prediction_time_adjustment_func
 
     def label(self, patient: Patient) -> List[Label]:
-        """Label all admissions with admission length > `self.long_time`"""
+        """
+        Label all admissions with admission length > `self.long_time`
+        Exclude admission if time of discharge or death < prediction time
```

Comment:
This is a neat change. I think we might actually want to make this universal (make it so that all predictions after death are discarded). Would be another PR, but what would your thoughts be about that?

Reply:
That is a good idea. I can't think of a scenario for which we wouldn't apply this rule.